config/ab_tests.yml

```diff
@@ -50,3 +53,6 @@ test:
   2021_10_mobile_search_panel_test:
     present: 1
     overlay: 0
+  2021_11_desktop_search_results_page_test:
+    present: 1
+    alternative: 0
```

Comment:
i would rather these names were a bit more descriptive `alternative` doesnt mean much to me, whereas `right` or `right_column` for example gives a bit more clue to whats going on